runs.py

```diff
@@ -107,6 +108,19 @@ def run_length(self) -> str:
             return f"{hours}:{minutes:>02}:{seconds:>02}"
         return f"{minutes:>02}:{seconds:>02}"
 
+    @property
+    def removals(self) -> list[str]:
+        neow_bonus_removals = self._data.get("neow_bonus_log", {}).get("cardsRemoved", [])
```

Comment:
I *think* I prefer to do `self.neow_bonus` for this, though it would need additional methods as well for these. In general, I want `NeowBonus` to be able to be slotted in as if it were a `NodeData` instance, so it should probably have similar methods. Maybe making them both inherit from the same class could be good, but that's outside of the scope of this PR. If you want to keep this line as-is, add a comment stating to add a property to `NeowBonus`.